package.json

```diff
@@ -7,7 +7,7 @@
   "main": "main.js",
   "scripts": {
     "crawl": "node ./cli/crawl-cli",
-    "test": "npm run lint && tsc && npm run unit",
```

Comment:
hm, doesn't this remove typechecking? I think we want typechecking. Or is this run somehow via lint now?

Reply:
Hm, I was sure typescript-eslint did typecheking, but apparently it needs [another config](https://typescript-eslint.io/getting-started/typed-linting/). I tried to enable it, but got a bunch of new errors (our code is messy), so I added tsc back here.